### DISCUSSION TITLE
Expand the disk after install

### DISCUSSION THREAD

**Discussion by frhulst** [community-scripts/ProxmoxVE#234] [Q&A] [answered]
I installed a ubuntu vm and i need to expand the disk and i remember on the old site there was a link to something on github of tteck showing some after install steps, however after the transition i cant find it

thanks in advance!!!

**Comment by tesla-76** [comment, marked as answer]
I do it this way : go to the VM then HARDWARE, select the disk to expand its size and click on the top menu DISK ACTION and then RESIZE.

**Reply by Node815** [reply]
To add to this, it's best to either shutdown the VM and resize (basically you have to calculate the desired capacity - current setting to get what number to put in.   Then, after this, start the VM.    Otherwise, it would just wait until the next shutdown and up.   I sometimes need to do this with my Home Assistant VM.
